src/main/java/com/spring/app/controllers/EmployeeController.java

```diff
@@ -11,45 +13,42 @@
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Date;
 import java.util.Locale;
 import java.util.Map;
 
+/**
+ * EmployeeController
+ *
+ * @author lyubov
+ */
 @Controller
 public class EmployeeController {
     @Autowired
     private EmployeeRepository employeeRepository;
 
     @GetMapping("employees")
-    public String main(Map<String, Object> model) {
-        Iterable<Employee> messages = employeeRepository.findAll();
-
-        model.put("employees", messages);
-
-        return "employee";
+    public ResponseEntity<Iterable<Employee>> main(Map<String, Object> model) {
+        Iterable<Employee> employees = employeeRepository.findAll();
+        return new ResponseEntity<>(employees, HttpStatus.OK);
     }
 
     @PostMapping("employees")
-    public String add(@RequestParam String birthday, @RequestParam String firstWorkDay, Map<String, Object> model) {
+    public ResponseEntity<Employee> add(@RequestParam String birthday, @RequestParam String firstWorkDay, Map<String, Object> model) {
         DateFormat df = new SimpleDateFormat("yyyy-mm-dd", Locale.ENGLISH);
         try {
             Employee employee = new Employee(df.parse(birthday), df.parse(firstWorkDay));
             employeeRepository.save(employee);
+            return new ResponseEntity<>(employee, HttpStatus.OK);
         } catch (ParseException e) {
             e.printStackTrace();
+            //todo remove stack to log! and find what response should i return if smth goes wrong
+            return new ResponseEntity<>( HttpStatus.BAD_REQUEST);
         }
-
-        Iterable<Employee> employees = employeeRepository.findAll();
-
-        model.put("employees", employees);
-
-        return "employee";
     }
 
-    @PostMapping("employees/filter")
```

Comment:
You don't need to have several filter methods. Just create a single method with an optional date param

Reply:
I have removed this method at all :) that's not a problem any more